src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -47,6 +47,21 @@
 logger = logging.getLogger("pyfluent.launcher")
 
 
+def _remove_unused_args(fluent_launch_mode: LaunchMode = None, **kwargs):
+    if fluent_launch_mode == LaunchMode.STANDALONE or LaunchMode.PIM:
+        for key in [
+            "start_container",
+            "container_dict",
+            "dry_run",
+            "scheduler_options",
+        ]:
+            kwargs.pop(key)
+    elif fluent_launch_mode == LaunchMode.CONTAINER:
+        for key in ["start_container", "scheduler_options"]:
+            kwargs.pop(key)
+    return kwargs
```

Comment:
Added to resolve codegen error.

![image](https://github.com/ansys/pyfluent/assets/106588300/64e7f3aa-bce2-48e0-a305-ef84292f78e1)